### ISSUE TITLE
Open in larger window / allow maximization 

### ISSUE THREAD

**Issue by AJolly** [aloneguid/bt#109]
BT Always launches in a small window.
 It also has maximization greyed out so I can't even maximize the window.

Could we please:
1)  allow maximization
2)  automatically pick a larger window size or remember our previous window size or something of that nature.
3)  enable wrapping for the profile list as well as the arguments and extra arguments.  (my params are right now around 500 chars)

![Screenshot 2024-12-09 181841](https://github.com/user-attachments/assets/79d89cdf-fbc6-4d0f-adbc-da9cf5532d09)

**Comment by aloneguid** [comment]
I won't give you everything, but there will be two things making it easier in the next patch - tab list popup and scrolling tabs:

![image](https://github.com/user-attachments/assets/670d94b5-9816-412d-935a-f5977ce7c6a0)

**Comment by AJolly** [comment]
Thanks that will help. Is there a reason maximization is always greyed out?

**Comment by aloneguid** [comment]
Not really, only for historical reasons. I have enabled this in the next patch, see if this will be fine.
